Source/SPDataImport.m

```diff
@@ -371,6 +373,16 @@ - (void)startSQLImportProcessWithFile:(NSString *)filename
  */
 - (void)importSQLFile:(NSString *)filename
 {
+	SPLog(@"Starting import....");
+	
+//	TODO: this is slowwwwwwww
```

Comment:
This should be removed before merging?

Reply:
Maybe, it's still TODO though. Probably the process should be to raise an issue rather than leave a TODO in there.

Reply:
Yeah, I think "TODO"s shouldn't be merged to master, either put a warning there or create an issue :)